TheAmazingAudioEngine/Utilities/AEIOAudioUnit.m

```diff
@@ -48,6 +49,69 @@ @interface AEIOAudioUnit ()
 #endif
 @end
 
+
+////////////////////////////////////////////////////////////////////////////////
+// sets nr of seconds between reports, 0 = no reporting
+#define AE_REPORT_TIME 2
+
+#if AE_REPORT_TIME
+
+static OSStatus notifyCallback(
+	void *							inRefCon,
+	AudioUnitRenderActionFlags *	ioActionFlags,
+	const AudioTimeStamp *			inTimeStamp,
+	UInt32							inBusNumber,
+	UInt32							inNumberFrames,
+	AudioBufferList * __nullable	ioData)
+{
+	static UInt64 startTime = 0;
```

Comment:
There can be multiple IO units in the same process, so these variables should rather be instance variables.